e2e/rbd_helper.go

```diff
@@ -40,6 +40,20 @@ var nbdResizeSupport = []util.KernelVersion{
 	}, // standard 5.3+ versions
 }
 
+// To use `io-timeout=0` we need
+// www.mail-archive.com/linux-block@vger.kernel.org/msg38060.html
+// nolint:gomnd // numbers specify Kernel versions.
+var nbdZeroIOtimeoutSupport = []util.KernelVersion{
+	{
+		Version:      5,
+		PatchLevel:   4,
+		SubLevel:     0,
+		ExtraVersion: 0,
+		Distribution: "",
+		Backport:     false,
+	}, // standard 5.4+ versions
```

Comment:
Didn't you mention that the CentOS 8.4 kernel was also good? That can probably be added as a backport kernel, and running the e2e tests on OpenShift with current RHEL CoreOS should then work with the right timeouts.

Reply:
Yes, we could, Do you know what kernel version does Centos 8.4 run?

Reply:
From here: https://en.wikipedia.org/wiki/CentOS
I see it as `4.18.0-305`

Will add the same, thanks Niels.